src/drawer.multicanvas.js

```diff
@@ -90,6 +99,14 @@ export default class MultiCanvas extends Drawer {
          * @type {number}
          */
         this.barRadius = params.barRadius || 0;
+
+        /**
+         * A helper to determine whether this uses split channels
```

Comment:
can you expand a little more in this docstring and mention it's related to the `splitChannels` option (and `channelColors` maybe as well)?

Reply:
this would also be a good public var IMO.

Reply:
Sure. I can make this public and improve the documentation.